core/src/main/scala/org/apache/spark/scheduler/ReplayListenerBus.scala

```diff
@@ -107,6 +107,16 @@ private[spark] class ReplayListenerBus extends SparkListenerBus with Logging {
         }
       }
     } catch {
+      case eofe: EOFException =>
+        // If the history event file is compressed and inprogress, the compressor will throw an
+        // EOFException if there is not enough to decompress a proper frame.  This indicates
+        // we're at the end of the file so we treat similarly to the JsonParseException case above.
+        if (!maybeTruncated) {
+          throw eofe
+        } else {
+          logWarning(s"Got EOFException from log file $sourceName" +
```

Comment:
Agree, the message doesn't add much value since this is expected behavior.  Will remove.